src/v2i-hub/MUSTSensorDriverPlugin/src/MUSTSensorDriverPlugin.cpp

```diff
@@ -0,0 +1,116 @@
+/**
+ * Copyright (C) 2024 LEIDOS.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+#include "MUSTSensorDriverPlugin.h"
+
+using namespace tmx::utils;
+using namespace std;
+
+namespace MUSTSensorDriverPlugin {
+	
+	MUSTSensorDriverPlugin::MUSTSensorDriverPlugin(const string &name): PluginClientClockAware(name)
+	{
+		mustSensorPacketReceiverThread = std::make_unique<tmx::utils::ThreadTimer>(std::chrono::milliseconds(5));
+		if (PluginClientClockAware::isSimulationMode()) {
+			PLOG(tmx::utils::logINFO) << "Simulation mode on " << std::endl;
+			SubscribeToMessages();
```

Comment:
Can we add a comment here that 'Only subscribe to timesync message when it is in simulation mode.'